designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/Sections.kt

```diff
@@ -318,7 +319,7 @@ internal fun SectionTitle(
     Text(
         modifier = modifier,
         text = title,
-        style = MaterialTheme.typography.titleLarge,
+        style = MaterialTheme.typography.titleLarge.copy(fontWeight = FontWeight.Medium),
```

Comment:
I think that titleLarge needs to be modified it's font weight, currently is:
`   titleLarge = TextStyle(
                fontSize = 22.sp,
                lineHeight = 28.sp,
                fontFamily = provideFontResource(),
                fontWeight = FontWeight.Normal,
                color = TextColor.OnSurface,
            ),`
And in the specs is 👍🏽 
![image](https://github.com/dhis2/dhis2-mobile-ui/assets/3330344/1b45f3c0-470b-4902-8866-6a29f6bef020)
So maybe you need to change to FontWeight.Medium in the typography , right?